docs/src/token.md

```diff
@@ -449,6 +458,17 @@ consistently valid Multisig accounts all Solana accounts holding a Account,
 Mint, or Multisig must contain enough SOL to be considered [rent
 exempt](https://docs.solana.com/implemented-proposals/rent)
 
+However note that there is currently a loophole to escape from the rent-exemption
+rule. So it isn't technically impossible to create spl-token Solana accounts
+which aren't actually rent-exempt. This loophole was unintentionally created
+because the token program is susceptible to sysvar spoofing. Specifically, there
+was an insufficient check for accounts passed as sysvar while executing in
+solana-program-sdk, which is statically linked into the token program.
```

Comment:
Can remove "while executing in
solana-program-sdk, which is statically linked into the token program." I think

Reply:
How about just: "It is possible to create SPL Token accounts that are not rent exempt by spoofing the Rent sysvar, since there are insufficient sysvar checks in the program. This could be abused to burn tokens by transferring tokens to a non-exempt Account that is subsequently rent-collected out of existence."